requirements/edx/base.txt

```diff
@@ -1061,9 +1144,9 @@ slumber==0.7.1
     #   edx-bulk-grades
     #   edx-enterprise
     #   edx-rest-api-client
-snowflake-connector-python==3.11.0
+snowflake-connector-python==3.12.0
     # via edx-enterprise
-social-auth-app-django==5.4.1
+social-auth-app-django==5.4.2
```

Comment:
This change will cause issues for large installations like edx.org for little benefit. 2U was planning on pinning this dependency to 5.4.1 until we're ready to handle faking the migrations.